purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -1844,7 +1844,8 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
         @JvmStatic
         @JvmSynthetic
         internal fun configure(
-            configuration: PurchasesConfiguration
+            configuration: PurchasesConfiguration,
+            overrideBillingAbstract: BillingAbstract? = null
```

Comment:
Note how this is actually part of the main flavor. However this method is internal, so it's not exposed to the public. So I think it's ok